src/sql/parts/connection/common/connectionManagement.ts

```diff
@@ -220,6 +220,8 @@ export interface IConnectionManagementService {
 
 	canChangeConnectionConfig(profile: ConnectionProfile, newGroupID: string): boolean;
 
+	getTabColorForUri(uri: string): string;
```

Comment:
we really need to refactor this class...it seems like the 'grab bag` for misc. (Not really related to this PR though)